examples/shapes/src/main.rs

```diff
@@ -14,173 +14,219 @@ use winit::{event::*, event_loop::ControlFlow};
 use winit::{event_loop::EventLoop, window::Window};
 
 // Simple struct to hold the state of the renderer
-pub struct RenderState<'s> {
+pub struct ActiveRenderState<'s> {
     // The fields MUST be in this order, so that the surface is dropped before the window
     surface: RenderSurface<'s>,
     window: Arc<Window>,
 }
 
-// Helper function that creates a Winit window and returns it (wrapped in an Arc for thread safety)
-fn create_window(event_loop: &winit::event_loop::EventLoopWindowTarget<()>) -> Arc<Window> {
-    Arc::new(
-        WindowBuilder::new()
-            .with_inner_size(LogicalSize::new(1044, 800))
-            .with_resizable(true)
-            .with_title("Vello Shapes")
-            .build(event_loop)
-            .unwrap(),
-    )
+enum RenderState<'s> {
+    Active(ActiveRenderState<'s>),
+    // Cache a window so that it can be reused when the app is resumed after being suspended
+    Suspended(Option<Arc<Window>>),
 }
 
 fn main() -> Result<()> {
-    let event_loop = EventLoop::new()?;
+    // Setup a bunch of state:
+
+    // The vello RenderContext which is a global context which lasts for the lifetime of the application
```

Comment:
```suggestion
    // The vello RenderContext which is a global context that lasts for the lifetime of the application
```

(Nit, mostly because of duplicate "which", not important...)